spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GcpContextAutoConfiguration.java

```diff
@@ -47,7 +48,12 @@
 @EnableConfigurationProperties(GcpProperties.class)
 public class GcpContextAutoConfiguration {
 
-	private static final String GCP_PUBSUB_SCOPE = "https://www.googleapis.com/auth/pubsub";
+	private static final String PUBSUB_SCOPE = "https://www.googleapis.com/auth/pubsub";
+
+	private static final String SQLADMIN_SCOPE =
+			"https://www.googleapis.com/auth/sqlservice.admin";
+
+	private static final List<String> SCOPES_LIST = ImmutableList.of(PUBSUB_SCOPE, SQLADMIN_SCOPE);
```

Comment:
Hum I'm not sure that's worth the trouble TBH... It's more complexity for the user with questionable benefit.

Unless a user wants to make sure that their app never does anything Pub/Sub, and they don't specify the scope? But then, I think they should just disable that permission on IAM instead, not providing a scope to achieve that would be the wrong way IMO.